api/src/main/java/io/strimzi/api/kafka/model/status/KafkaBridgeStatus.java

```diff
@@ -19,13 +19,15 @@
         builderPackage = "io.fabric8.kubernetes.api.builder"
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "conditions", "observedGeneration", "httpAddress" })
+@JsonPropertyOrder({ "conditions", "observedGeneration", "httpAddress", "restApiPort" })
 @EqualsAndHashCode
 public class KafkaBridgeStatus extends Status {
     private static final long serialVersionUID = 1L;
 
     private String httpAddress;
 
+    private int restApiPort;
+
     @Description("The HTTP address at which external client applications can access the Kafka Bridge.")
     public String getHttpAddress() {
```

Comment:
I'm confused about what this is. Is the httpAddress the host part of the address, and the RestApiPort the port? If so then the naming is confusing and we should change `httpAddress` → `restApiHost` or similar.

In any case can you clarify in the `@Description` what these properties are?